src/main/scala/org/ergoplatform/nodeView/wallet/requests/AssetIssueRequest.scala

```diff
@@ -0,0 +1,69 @@
+package org.ergoplatform.nodeView.wallet.requests
+
+import io.circe.syntax._
+import io.circe.{Decoder, Encoder, HCursor, Json}
+import org.ergoplatform.ErgoBox.{R4, R5, R6}
+import org.ergoplatform.api.ApiCodecs
+import org.ergoplatform.nodeView.wallet.{ErgoAddress, ErgoAddressEncoder}
+import org.ergoplatform.settings.ErgoSettings
+import org.ergoplatform.{ErgoBox, ErgoBoxCandidate}
+import scorex.core.transaction.box.Box.Amount
+import sigmastate.Values.{IntConstant, StringConstant}
+
+/**
+  * Request for issuing new asset of Ergo Asset Standard 1.
```

Comment:
Do we have any agreement about specification naming and numbering?

Reply:
@mike-aksarin, No, I think any other option could be discussed. This name is just to make the point of point clear.